docs/arm64.md

```diff
@@ -0,0 +1,64 @@
+# How to build and run Cloud-hypervisor on Arm64
+
+Cloud-hypervisor is partially enabled on AArch64 architecture.
+Although all features are not ready yet, you can begin to test Cloud-hypervisor on a Arm64 host by following this guide.
+
+## Prerequisites
+
+On Arm64 machines, Cloud-hypervisor depends on an external library `libfdt-dev` for generating Flatted Device Tree (FDT).
```

Comment:
Typo: `Flatted` should be `Flattened`

Reply:
@michael2012z Can you please fix this in a subsequent PR.

Reply:
Thanks, @sboeuf , @rbradford . I will fix it in my next PR.